apps/rolodex/views/partials/app/sidebar_temp.ejs

```diff
@@ -0,0 +1,42 @@
+<div class="max-md:flex max-md:justify-center max-md:items-center avatar online placeholder max-md:mx-20">    <div class="bg-neutral-focus text-neutral-content rounded-full w-32">
```

Comment:
I don't see a need to make a new "sidebar_temp" file, why can it be in the "sidebar" itself?
1. This new file tries to call functions and modals defined in the previous file - sacrificing readability.
2. And since this new file will not be used anywhere else, modularity has no benefit.

Reply:
yes, you are correct. I have removed that file